frontend/src/lib/taxonomy.tsx

```diff
@@ -20,21 +43,11 @@ const eventToPersonProperties = new Set([
     '$os_version',
     '$referring_domain',
     '$referrer',
-    // campaign params - automatically added by posthog-js here https://github.com/PostHog/posthog-js/blob/master/src/utils/event-utils.ts
-    'utm_source',
-    'utm_medium',
-    'utm_campaign',
-    'utm_content',
-    'utm_name',
-    'utm_term',
-    'gclid',
-    'gad_source',
-    'gbraid',
-    'wbraid',
-    'fbclid',
-    'msclkid',
```

Comment:
This was actually missing a bunch of newer addition to `posthog-js` which `CAMPAIGN_PROPERTIES` now includes – another kind of inconsistency silently arising over time

Reply:
I've added a test to make sure that everything in this list has a definition